src/main/java/com/gildedrose/infraestructure/ItemsService.java

```diff
@@ -0,0 +1,66 @@
+package com.gildedrose.infraestructure;
```

Comment:
Ahora a la madre del cordero....
Como has hecho el programa ,contradice el primer y segundo principios SOLID.
1) Single-responsability principle: Una clase solo debería tener una razón para cambiar, lo que quiere decir que solo tiene que tener una función, no varias. Tu clase ItemsService se encarga de varias cosas, y si en un futuro hubieran más tipos de items con sus "reglas" específicas , esta clase iría creciendo y creciendo.
2) Open-Close  principle : Open for Extension close to modification: Deberías poder añadir funcionalidades a una clase sin tener que modificarla. Una manera para añadir funcionalidades sin modificar el código que ya tienes y que ya sabes que funciona bien, sería añadiendo Interfaces.